app/uk/gov/hmrc/trackingconsentfrontend/assets/src/domain/userPreferenceFactory.ts

```diff
@@ -23,26 +26,36 @@ const setPreferences = preferencesIn => {
   })
 }
 
+const sanitizePreferences = preferences => {
```

Comment:
We tend to avoid Americanisms, `sanitisePreferences` would be more in keeping with other projects I've worked on.  (for example GDS use `colour` constants rather than `color`.